project/build.scala

```diff
@@ -345,6 +348,7 @@ object Dependency {
   val akkaActor = "com.typesafe.akka" %% "akka-actor" % V.Akka
   val akkaSlf4j = "com.typesafe.akka" %% "akka-slf4j" % V.Akka
   val akkaStream = "com.typesafe.akka" %% "akka-stream" % V.Akka
+  val akkaHttp = "com.typesafe.akka" %% "akka-http-experimental" % V.Akka
```

Comment:
http-core is marked stable, but the higher API (which happens to be where
marshaling is) is still undergoing changes. Marshaling is quite close to
what it was in spray, so I'm not that concerned.
On Tue, Jul 12, 2016 at 9:24 AM Ken Sipe <notifications@github.com> wrote:

> In project/build.scala
> <https://github.com/mesosphere/marathon/pull/4062#discussion_r70471915>:
>
> > @@ -345,6 +348,7 @@ object Dependency {
> >    val akkaActor = "com.typesafe.akka" %% "akka-actor" % V.Akka
> >    val akkaSlf4j = "com.typesafe.akka" %% "akka-slf4j" % V.Akka
> >    val akkaStream = "com.typesafe.akka" %% "akka-stream" % V.Akka
> > +  val akkaHttp = "com.typesafe.akka" %% "akka-http-experimental" % V.Akka
>
> I'm guessing you have enough experience with akka-http that you have
> confidence in not being concerned with the "-experimental" suffix :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4062/files/5ffaa7e4c7657fc609814fd932a7343acfb909f0#r70471915>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAweZr-Iqv6_aSYbwIRKTE9DoD9whyZTks5qU7-wgaJpZM4JDjhV>
> .
>